packages/react/src/components/ListBox/next/ListBoxSelection.tsx

```diff
@@ -10,30 +10,32 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import { Close } from '@carbon/icons-react';
 import { usePrefix } from '../../../internal/usePrefix';
+import type { TFunc, TranslateWithId } from '../../../types/common';
 
+// TODO: Why is this code comment here?
```

Comment:
Listbox components were initially internal-only and not exported. They weren't (and still aren't) documented in storybook, so this kind of explanatory documentation was initially included for maintainers' sake. Similar to `packages/react/src/components/ListBox/README.md`

This is all before my time, but that's my understanding of why all the ListBox components have these.